src/text_file_functions.cpp

```diff
@@ -715,3 +715,84 @@ QString AOApplication::get_casing_can_host_cases()
   QString result = configini->value("casing_can_host_cases", "Turnabout Check Your Settings").value<QString>();
   return result;
 }
+
+int AOApplication::get_saved_ooc_format()
+{
+  QString value_string = configini->value("saved_ooc_format", "Plain text (default)").value<QString>();
+  format_ooc result;
+  if (value_string == "Raw HTML")
+    result = HTML;
+  #if (QT_VERSION >= QT_VERSION_CHECK(5, 14, 0))
```

Comment:
This is a weird check to be making everywhere, and you may wish to move it somewhere else and replace it with:
```suggestion
#ifdef MARKDOWN
```

Reply:
Implemented this, thank you